shared/chat/conversation/input-area/normal/platform-input-animation.native.tsx

```diff
@@ -56,6 +56,7 @@ export const runTiming = (
       set(config.toValue, dest),
       startClock(clock),
     ]),
+    // @ts-ignore
```

Comment:
there is a new api for reanimated that we can adopt later. the old api isn't typed so we just ignore it